pkg/e2e/e2e.go

```diff
@@ -144,9 +145,20 @@ func beforeSuite() bool {
 		}
 
 		var kubeconfigBytes []byte
-		if kubeconfigBytes, err = provider.ClusterKubeconfig(viper.GetString(config.Cluster.ID)); err != nil {
+		clusterConfigerr := wait.PollImmediate(2*time.Second, 5*time.Minute, func() (bool, error) {
+			kubeconfigBytes, err := provider.ClusterKubeconfig(viper.GetString(config.Cluster.ID))
```

Comment:
This should be `=` not `:=`

By making it `:=` it creates a `kubeconfigBytes` variable only within the scope of the inner PollImmediate function and not honoring the defined variable on line 147

Reply:
Pr updated good catches!